features/topic_id_subtopics.feature

```diff
@@ -0,0 +1,158 @@
+Feature: Behaviour of application when doing the GET /topics/{id}/subtopics endpoint, using a stripped down version of the database
```

Comment:
same in this file, the db setup steps can be moved into one background block

Reply:
Done